v-next/hardhat/templates/mocha-ethers/hardhat.config.ts

```diff
@@ -7,6 +7,17 @@ import HardhatNetworkHelpers from "@ignored/hardhat-vnext-network-helpers";
 const config: HardhatUserConfig = {
   plugins: [HardhatMochaTestRunner, HardhatEthers, HardhatNetworkHelpers],
   solidity: "0.8.24",
+  defaultNetwork: "hardhat",
+  networks: {
+    "local-base": {
+      chainId: 8453,
+      type: "edr",
+      chainType: "optimism",
+      gas: "auto",
```

Comment:
This seems required for the moment, which is unexpected. We should have defaults here.